artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireProtocolManager.java

```diff
@@ -154,6 +154,11 @@ public OpenWireProtocolManager(OpenWireProtocolManagerFactory factory, ActiveMQS
       if (cc != null) {
          cc.addClusterTopologyListener(this);
       }
+      if (supportAdvisory) {
```

Comment:
I wouldn't make it dependent on supportAdvisory.

Say you have two acceptors.. different IPs.. you will have this called twice.

I would always add the filter, and check if the filter is not part of the expression before you called it.

Reply:
I'm getting a failure with the last changes...